### PR TITLE
craterun deployment for staging added

### PR DESCRIPTION
**Pull request by lemoninfinex**

### CODE REVIEW

File: craterun/mainnets/infinex-craterun-base.toml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+version = "1.0.0"
+description = "Smart Contracts used for the Infinex Craterun Campaign"
+
+# --chain-id 8453
+
+[var.Deploy]
+CRATERUN_MULTI_SIG = "0x7299aF061A210157f7AcaEaDa22534ae3b72C956"
+CRATERUN_DEPLOYER = "0x3F768E3aC5eE75A2F39CFD7d72Eb3a40Bddb5ecB"
+CRATERUN_KEEPER= "0xc487D6ADF3de4D469E52804902b89c9427c74c2a"
+CRATE_POINTS_NAME = "Infinex CratePoints"
```
Comment (axefinaxe) [inline craterun/mainnets/infinex-craterun-base.toml:12]: surprised this name doesn't clash...

File: craterun/staging/infinex-craterun-base.toml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+# ---------- Initializers ---------- #
+
+[invoke.CratePointsInitialize]
+target = ["InfinexCraterun.CratePointsProxy"]
+func = "initialize"
+args = ["<%= settings.CRATERUN_DEPLOYER %>", "<%= settings.CRATE_POINTS_NAME %>"]
+
+# ---------- CratePoints Minters ---------- #
+
+[invoke.CratePointsSetMinterForDeployer]
```
Comment (rayInfinex) [inline craterun/staging/infinex-craterun-base.toml:39]: remove this

File: craterun/testnets/infinex-craterun-base-sepolia.toml [hunk 3]
```diff
@@ -22,16 +20,54 @@ options.ENTROPY_PROVIDER = "0x6CC14824Ea2918f5De5C2f75A9Da968ad4BD6344" # Pyth E
...
 [invoke.CratePointsInitialize]
 target = ["InfinexCraterun.CratePointsProxy"]
 func = "initialize"
 args = ["<%= settings.CRATERUN_DEPLOYER %>", "<%= settings.CRATE_POINTS_NAME %>"]
-depends = ["clone.InfinexCraterun"]
\ No newline at end of file
+
+# ---------- CratePoints Minters ---------- #
+
+[invoke.CratePointsSetMinterForDeployer]
```
Comment (rayInfinex) [inline craterun/testnets/infinex-craterun-base-sepolia.toml:39]: same here

Comment (lemoninfinex) [reply craterun/testnets/infinex-craterun-base-sepolia.toml:39]: no I won't remove it from here

**Review by rayInfinex — APPROVED** [review]

**Pull request merged**